### DISCUSSION TITLE
Cryptocurrency sell, and buy pages type

### DISCUSSION THREAD

**Discussion by mrekh** [schemaorg/schemaorg#3100] [Q&A] [answered]
Is the "Service" schema applicable to cryptocurrency sell, and buy pages, such as "Bitcoin sell, and buy", "ETH sell, and buy", etc.?
Is it possible to consider them a service page?

**Comment by thadguidry** [comment, marked as answer]
They would be a provider and making an Offer.  On the Buy or Sell pages, use Offer or Demand.  Also look at our Actions such as BuyAction etc. Which might be useful also.  If you need specific help, I am available for consulting further.  But first take a look specifically at the JSON-LD first example at the bottom of https://schema.org/Service
